recipes/chemfiles-lib/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.6.0" %}
+{% set sha256 = "4e43dbe1b1d4c3a2eae923fecc3452d2b207efc1d511f6683164d51d22b0255f" %}
+
+package:
+  name: chemfiles-lib
+  version: {{ version }}
+
+about:
+  home: http://chemfiles.github.io
+  license: MPL-v2.0
+  summary: Modern library for chemistry file reading and writing
+  doc_url: http://chemfiles.github.io/chemfiles/latest/
+  dev_url: https://github.com/chemfiles/chemfiles
+
+build:
+  script:
+    - export MACOSX_DEPLOYMENT_TARGET=""  # [osx]
+    - cmake -DCMAKE_INSTALL_PREFIX=$PREFIX -DENABLE_NETCDF=ON .
+    - cmake --build . --config release --target install
+    - cmake -DBUILD_SHARED_LIBS=ON .
+    - cmake --build . --config release --target install
```

Comment:
For more complex scripts, we like to break them out into `build.sh` and `bld.bat`.